tests/nnstreamer_filter_llamacpp/unittest_filter_llamacpp.cc

```diff
@@ -60,6 +60,12 @@ class NNStreamerFilterLlamaCppTest : public ::testing::Test
    */
   NNStreamerFilterLlamaCppTest ()
   {
+    pipeline = nullptr;
+    appsrc = nullptr;
+    appsink = nullptr;
+    tensor_filter = nullptr;
+    model = nullptr;
+    pipeline_str = nullptr;
```

Comment:
With this change, I suggest clearly setting null when releasing objects in TearDown().

```
g_clear_pointer (&pipeline, gst_object_unref);
g_clear_pointer (&appsrc, gst_object_unref);
g_clear_pointer (&appsink, gst_object_unref);
g_clear_pointer (&tensor_filter, gst_object_unref);
g_clear_pointer (&pipeline_str, g_free);
g_clear_pointer (&model, g_free);
```